### DISCUSSION TITLE
Write multifab and imultifab with different types in the same file

### DISCUSSION THREAD

**Discussion by avergnaud002** [AMReX-Codes/amrex#3336] [Q&A] [answered]
Hello,

I want to write in the same file different multifab and/or imultifab with different types (typically cell-centered variables and face-centered variables). Is there a simple way to do that ?

Notes : I use VisIt as vizualisation tool, and my code is written in fortran (so I use the Fortran Interface of AMReX).

Thanks
Alban

**Comment by WeiqunZhang** [comment, marked as answer]
I don't think VisIt supports non-cell-centered data in AMReX plotfiles. So you need to convert the data to cell centers and store them in a single MultiFab.

**Reply by avergnaud002** [reply]
thank you for your answer.
